docs/sdk-and-tools/indices/accounts.md

```diff
@@ -17,14 +17,17 @@ The `_id` field of this index is represented by a bech32 encoded address.
-| balance    | The balance field holds the amount of EGLD the address possesses. It is a string that also includes the number of decimals. Example: "1500000000000000000" (equivalent to 1.5 EGLD). |
-| balanceNum | The balanceNum field holds the amount of EGLD the address possesses, in a numeric format. Example: 1.5.                                                                              |
-| nonce      | The nonce field represents the sequence number of the address.                                                                                                                       |
-| shardID    | The shardID field represents the shard where the address belongs to, based on its bytes.                                                                                             |
-| timestamp  | The timestamp field represents the last moment when the address balance was changed.                                                                                                 |
+| Field            | Description                                                                                                                                                                          |
+|------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| address          | The address field holds the address in a bech32 encoding. Should be equal to the _id field.                                                                                          |
+| balance          | The balance field holds the amount of EGLD the address possesses. It is a string that also includes the number of decimals. Example: "1500000000000000000" (equivalent to 1.5 EGLD). |
+| balanceNum       | The balanceNum field holds the amount of EGLD the address possesses, in a numeric format. Example: 1.5.                                                                              |
+| nonce            | The nonce field represents the sequence number of the address.                                                                                                                       |
+| shardID          | The shardID field represents the shard where the address belongs to, based on its bytes.                                                                                             |
+| timestamp        | The timestamp field represents the last moment when the address balance was changed.                                                                                                 |
+| developerRewards | The developerRewards represents the fees that were accumulated after all the smart contract calls that can be claimed by the owner.                                                  |
```

Comment:
The developerRewards represents the fees that were accumulated after all the smart contract calls. They can be claimed by the owner.

Reply:
done